src/Core/Domain/ApiAccess/Command/EditApiAccessCommand.php

```diff
@@ -118,4 +120,16 @@ public function setScopes(?array $scopes): self
 
         return $this;
     }
+
+    public function getLifetime(): ?int
+    {
+        return $this->lifetime;
+    }
+
+    public function setLifetime(?int $lifetime): self
```

Comment:
Maybe we could add some comment to explain if the value is expected in seconds or milliseconds